### PR TITLE
[14.0][FIX] sale_restricted_qty: remove post-migration junk

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@rousseldenis I guess this is worth cleaning as well

File: sale_restricted_qty/__manifest__.py [hunk 1]
```diff
@@ -6,8 +6,7 @@
     "version": "14.0.1.1.2",
     "category": "Sales Management",
     "author": "Akretion, Odoo Community Association (OCA)",
-    "contributors": ["Ashish Hirpara"],
-    "maintainers": ["ashishhirapara"],
+    "maintainers": ["alexey-pelykh"],
```
Comment (rousseldenis) [inline sale_restricted_qty/__manifest__.py:9]: Please don't remove the current one. Add yourself.

**Review by pedrobaeza — APPROVED** [review]

File: sale_restricted_qty/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply sale_restricted_qty/__manifest__.py:9]: I think that one is unresponsive, seeing the previous PRs, so why keeping it?

Comment (alexey-pelykh) [reply sale_restricted_qty/__manifest__.py:9]: That's my thought as well, there's no issue re-adding if anything changes. I would assume based on activity a lot of module maintainers (me including on some modules) should've been removed.

**Review by rousseldenis — APPROVED** [review]

File: sale_restricted_qty/__manifest__.py (hunk 1 above)
Comment (rousseldenis) [reply sale_restricted_qty/__manifest__.py:9]: Yes but sometimes, user is simply out of order but want to just maintain one module.

Comment (alexey-pelykh) [reply sale_restricted_qty/__manifest__.py:9]: Well, the person will have some time to object while 5 days cooldown time is there.

**Review by marcelsavegnago — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
I guess let's merge?

**Comment by alexey-pelykh** [comment]
The opposition time I guess is out - please merge

**Comment by alexey-pelykh** [comment]
@OCA/crm-sales-marketing-maintainers can we please merge?

**Pull request merged**